packages/libs/error-stack/src/fmt/mod.rs

```diff
@@ -1036,9 +1035,27 @@ fn debug_frame(root: &Frame, ctx: &mut HookContextImpl, prefix: &[&Frame]) -> Ve
             body.reverse();
             let body = debug_attachments(
                 Some(loc),
-                ctx,
-                (len == 1 && sources.is_empty()) || idx > 0,
+                // This makes sure that we use `\-` instead of `-`,
```

Comment:
Please correct me if I'm wrong. I know we spoke about this, but didn't we change `\-` to `|-`?

Reply:
That's correct. This is just in the documentation because I was too lazy to type 

> // This makes sure that we use `╰─` instead of `├─`

Will change.

(writing docs w/ unicode is just pain and suffering)

Reply:
Great, thansk!